### DISCUSSION TITLE
Uninstall @adonisjs/shield

### DISCUSSION THREAD

**Discussion by fenibofubara** [adonisjs/core#4132] [Help] [answered]
Hi guys, so i am trying to uninstall @adonisjs/shield after i installed it in the API server application i am building. I later learned that @adonisjs/shield' is not needed when building an API server.

I am getting this error each time I try to start the server. I have  uninstall @adonisjs/shield severally

   Error: Cannot find module '@adonisjs/shield'
   Require stack:
   - /home/fenibo/Desktop/03/O3XNodeServer/noop.js

Help, please

**Comment by McSneaky** [comment, marked as answer]
You need to also remove Shield from list of providers in `.adonisrc.json` and `tsconfig.json`. After that it's also safe to remove `config/shield.ts`
